api/src/repositories/attachment-repository.ts

```diff
@@ -369,6 +369,50 @@ export class AttachmentRepository extends BaseRepository {
     return response.rows;
   }
 
+  /**
+   * SQL query to get a single attachment for a single survey.
+   *
+   * @param {number} surveyId
+   * @param {number} attachmentId
+   * @return {*}  {Promise<ISurveyAttachment>}
+   * @memberof AttachmentRepository
+   */
+  async getSurveyAttachmentById(surveyId: number, attachmentId: number): Promise<ISurveyAttachment> {
+    defaultLog.debug({ label: 'getProjectAttachmentById' });
+
+    const sqlStatement = SQL`
+      SELECT
+        survey_attachment_id,
+        uuid,
+        file_name,
+        file_type,
+        title,
+        description,
+        create_date,
+        update_date,
+        create_date,
+        file_size,
+        key
+      FROM
+        survey_attachment
+      WHERE
+        survey_attachment_id = ${attachmentId}
+      AND
+        survey_id = ${surveyId};
+    `;
+
+    const response = await this.connection.sql<ISurveyAttachment>(sqlStatement);
+
+    if (!response.rows) {
```

Comment:
rows will always be truthy. So we should either remove this, or change the assertion to rowCount === 1